Добавлены новые классы

kafi40 opened the pull request:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

kafi40 committed:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

kafi40 committed:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

kafi40 committed:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

kafi40 committed:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

PolnySkvorets reviewed (commented):
Привет. По требованиям задачи не хватает тестов

kafi40 committed:
Добавить модель User
Добавить модель Film
Добавить контроллер UserController
Добавить контроллер FilmController
Добавить валидатор минимальной даты
Создать аннотацию для минимальной даты
Добавить логирование

kafi40 committed:
Добавить тесты для контроллеров

PolnySkvorets reviewed (approved):
Принято
